content/applications/productivity/spreadsheet/filters.rst

```diff
@@ -0,0 +1,205 @@
+   :alt: The filter options that appear after clicking the filter icon.
+
+Below are all the various options that can appear in the :guilabel:`Add a new filter...` list.
+
+Once the information is entered for a filter, click :guilabel:`Save` to save the filter
+configuration. Once saved, the filter appears beneath the :guilabel:`Filters` heading, and above the
+:guilabel:`Add a new filter...` section, and is ready to use. Once configured, the specific
+parameters within the filter can be selected, and the information is then presented.
+
+.. _filters/date:
+
+Date
+----
+
+This filter selection organizes and presents data in the spreadsheet that are based on a specific
+time period.
+
+First, the :guilabel:`Filter properties` must be configured. These settings determine what specific
+data is presented.
+
+Select this type of filter if the data should be filtered by specific time periods.
+
+Populate the following fields presented in the date filter:
+
+- :guilabel:`Label`: enter a name for the specific filter. This filter name appears when this
+  spreadsheet is opened in the future, and is available until the configured filter is deleted.
+- :guilabel:`Time range`: select the desired time frame to present. Options are either:
+
+  - :guilabel:`Month / Quarter`: presents data filtered by either a specific quarter or month.
+  - :guilabel:`Relative Period`: presents data for preconfigured time periods, other than a month
+    or quarter, such as :guilabel:`Year To Date`, or :guilabel:`Last 30 Days`.
+  - :guilabel:`From / To`: presents data that spans a specific time period **other than** a quarter
+    or month, such as the specific dates of an event, promotion, or management time period.
+
+- :guilabel:`Default value`: this selection appears **only** if either :guilabel:`Month / Quarter`
+  or :guilabel:`Relative Period` is selected for the :guilabel:`Time range`. The option presented is
+  to :guilabel:`Automatically filter (the data) on the current period`. Tick the checkbox to
+  activate this option. When activated, a drop-down menu appears. Select the time period to present,
+  either :guilabel:`Month`, :guilabel:`Quarter`, or :guilabel:`Year`.
+- :guilabel:`Field matching`: the spreadsheet can be further filtered down, if desired. It is
+  possible to specify which fields on the spreadsheet to apply the filter on. When entering a value
+  for a :guilabel:`Date` filter, two fields must be populated: the :guilabel:`Date Field` and the
+  :guilabel:`Period Offset`.
+
+When all the information is entered on the form, click the :guilabel:`Save` button.
+
+.. image:: filters/properties.png
+   :align: center
+   :alt: The filter panel with all the filter info entered.
+
+Once a filter is configured and saved, it appears beneath the :guilabel:`Filters` heading, and above
+:guilabel:`Add a new filter...`. Beneath the skill label, a drop-down menu appears. This allows to
+filter the data by the set :guilabel:`Time rage` set for the filter.
```

Comment:
I think this section could use an image, just because there are a few different location cues that make it a little confusing (below this, above this). 
Also, I am a little confused on this part, could use a bit of clarification- "This allows to
filter the data by the set :guilabel:`Time rage` set for the filter"